src/Planner/PlannerJoinTree.cpp

```diff
@@ -684,6 +693,40 @@ JoinTreeQueryPlan buildQueryPlanForTableExpression(QueryTreeNodePtr table_expres
         {
             if (!select_query_options.only_analyze)
             {
+                auto storage_merge_tree = std::dynamic_pointer_cast<MergeTreeData>(storage);
+                if (storage_merge_tree && query_context->canUseParallelReplicasOnInitiator()
+                    && settings.parallel_replicas_min_number_of_rows_per_replica > 0)
+                {
+                    ActionDAGNodes filter_nodes;
+                    if (table_expression_query_info.filter_actions_dag)
+                        filter_nodes.nodes = table_expression_query_info.filter_actions_dag->getOutputs();
```

Comment:
I'm not sure if we need to pass all filters here or we need to filter them more, like `getFiltersForPrimaryKeyAnalysis` does for the old interpreter.

Reply:
I'm not sure whether additional table filters setting is supported for the Analyzer...